libs/util/shared/src/lib/models/search.model.ts

```diff
@@ -46,6 +46,7 @@ export interface MetadataLinkValid {
   protocol?: string
   format?: string
   description?: string
+  label?: string
```

Comment:
why not

Reply:
Should not occur in practice I guess, but I put it optional as `name` and `description` are optional, as well